great_expectations/cli/docs.py

```diff
@@ -121,7 +129,37 @@ def docs_list(directory):
                 event="cli.docs.list",
                 success=False
             )
+@docs.command(name="clean")
+@click.option(
+    '--directory',
+    '-y',
+    default=None,
+    help="Clean data docs"
+)
+def clean_data_docs(directory, site_name=None):
+    """
+    Delete data docs
 
+    :param directory
+    :param site_name
+    """
```

Comment:
Just a heads up, if this is a click command, then your docstring should be the help text you want for the command, not a normal Python docstring. :)

Reply:
Yup @mzjp2 Updated.